src/clients/javascript/README.md

```diff
@@ -31,6 +31,9 @@ yarn add pizzly-js
 The `connect` method lets you trigger an OAuth-dance. On success, Pizzly returns an `authId` that acts as a unique identifier of the authentication process.
 
 ```js
+// Import pizzly-js package
+const Pizzly = require('pizzly-js')
```

Comment:
```suggestion
import Pizzly from 'pizzly-js'
```

Reply:
@tanguyantoine you're the JS expert, just let me know which one is the best. I'm using the suggested one in the main README.md